src/components/Admin/index.jsx

```diff
@@ -46,6 +46,21 @@ class Admin extends React.Component {
     return hasAnalyticsData;
   }
 
+  hasEmptyData() {
+    const {
+      numberOfUsers,
+      courseCompletions,
+      enrolledLearners,
+    } = this.props;
+    let hasEmptyData = true;
+
+    [courseCompletions, enrolledLearners, numberOfUsers].forEach((item) => {
+      hasEmptyData = item === 0 && hasEmptyData;
+    });
+
+    return hasEmptyData;
```

Comment:
might read better with `.every` (mdn)[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every]

```
return [courseCompletions, enrolledLearners, numberOfUsers].every((item) => item === 0);
```

Reply:
If we change to `.every` here, we may want to do it [here](https://github.com/edx/edx-portal/blob/fef1012df6974412c3f1063841a5215110aded51/src/components/Admin/index.jsx#L42) as well. #consistency

Reply:
i'm not sure we can use .every there because it is or'ing instead of and'ing.

Reply:
If thats the case we could use `.some` instead of `.every` but it looks to me like we're checking that every one of those values is zero?

Reply:
ah i meant for the other place adam linked - for this case it makes sense

Reply:
Yeah, you're right. Sorry about that :) That being said, perhaps `.some` would be of use there. Feel free to change it if you'd like.